src/redisraft.c

```diff
@@ -549,16 +549,65 @@ static bool handleInterceptedCommands(RedisRaftCtx *rr,
     return false;
 }
 
+static bool AskingHandleCommand(RedisRaftCtx *rr, RedisModuleCtx *ctx, RaftRedisCommandArray *cmds)
```

Comment:
I added a comment about style and I know it's not consistent in the codebase (I did bunch of mistakes) but let's try to follow the common style:
- Exported functions start with uppercase letter and a common prefix: 
   - RaftLogCreate(), RaftLogOpen() etc.
   - NodeAddrParse(), NodeAddrEqual() etc.
- Static functions start with lowercase letter. 
- Exception to this: common.c functions start with lowercase. (That's just the reality). So, these are just short helper functions . Probably, it's okay we keep it that way, start with lowercase letter without a common prefix as they are unrelated with each other anyway.

Reply:
in this case, isn't your comment consistent with this?

Reply:
Sorry I wasn't clear.  Yeah both are same.  The other comment(https://github.com/RedisLabs/redisraft/pull/347#discussion_r889877447) is consistent with this.